### PR TITLE
Fix #928: Switch CodeQL analysis to a new parent action from wultra-i…

### LINKED ISSUE #928: Switch CodeQL analysis to a new parent action from wultra-infrastructure
By solving https://github.com/wultra/wultra-infrastructure/pull/29 it is handy to use a new Github action parent for CodeQL analysis.

### PR DESCRIPTION
**Pull request by jandusil**
…nfrastructure

- Change code-analysis workflow, so it uses the new action parent from wultra-infrastructure

### CODE REVIEW

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -18,66 +7,15 @@ on:
...
-
-      # - run: |
-      #   echo "Run, Build Application using script"
-      #   ./location_of_script_within_repo/buildscript.sh
-
-      - name: Perform CodeQL Analysis
-        uses: github/codeql-action/analyze@v2
-        with:
-          category: "/language:${{matrix.language}}"
+  CodeQL_Analysis:
```
Comment (zcgandcomp) [inline .github/workflows/codeql-analysis.yml:13]: lower case with dashes, let's keep similar naming as in other actions

File: .github/workflows/codeql-analysis.yml [hunk 2]
```diff
@@ -1,14 +1,3 @@
...
-# or to provide custom queries or build logic.
-#
-# ******** NOTE ********
-# We have attempted to detect the languages in your repository. Please check
-# the `language` matrix defined below to confirm you have the correct set of
-# supported CodeQL languages.
-#
 name: "CodeQL"
 
 on:
```
Comment (zcgandcomp) [inline .github/workflows/codeql-analysis.yml:3]: I would include enable manual start - for convenience if anybody wants to invoke it manually

```
  workflow_dispatch:
    branches:
      - 'develop'
```

File: .github/workflows/codeql-analysis.yml [hunk 3]
```diff
@@ -18,66 +7,15 @@ on:
     # The branches below must be a subset of the branches above
     branches: [ develop ]
   schedule:
-    - cron: '0 2 * * 4'
+    - cron: '0 2 * * 5'
```
Comment (zcgandcomp) [inline .github/workflows/codeql-analysis.yml:10]: I would keep Thursday. Was there any particular reason to switch to Friday? But this is just minor technical detail.

**Review by zcgandcomp — CHANGES_REQUESTED** [review]

File: .github/workflows/codeql-analysis.yml (hunk 3 above)
Comment (jandusil) [reply .github/workflows/codeql-analysis.yml:10]: Apologies, this was changed during the testing phase to trigger the check

**Review by zcgandcomp — APPROVED** [review]
Looks OK.

**Pull request merged**